hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -632,6 +632,21 @@ public FileSystemViewStorageConfig getClientSpecifiedViewStorageConfig() {
     return clientSpecifiedViewStorageConfig;
   }
 
+  /**
+   * Commit call back configs.
+   */
+  public boolean writeCommitCallbackOn() {
+    return Boolean.parseBoolean(props.getProperty(HoodieWriteCommitCallbackConfig.CALLBACK_ON));
+  }
+
+  public String getCallbackType() {
```

Comment:
IMO, this property is not necessary. We can only depend on `getCallbackClass ` to specify the detailed implementation. This is a simple **SPI** mode.  Otherwise, it causes two entry points in the factory you implemented. WDYT?

Reply:
> IMO, this property is not necessary. We can only depend on `getCallbackClass ` to specify the detailed implementation. This is a simple **SPI** mode. Otherwise, it causes two entry points in the factory you implemented. WDYT?

done